extensions/vscode/src/test/runTest.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * This file downloads, unzipps, and launches VS Code with extension test parameters.
+ *
+ * @see https://code.visualstudio.com/api/working-with-extensions/testing-extension#the-test-script
+ * @see https://code.visualstudio.com/api/working-with-extensions/testing-extension#custom-setup-with-vscodetestelectron
+ * @see https://github.com/prettier/prettier-vscode/blob/main/src/test/runTests.ts
+ */
+
+import * as path from "path";
+import { runTests } from "@vscode/test-electron";
+
+async function main() {
+  try {
+    const extensionDevelopmentPath = path.resolve(__dirname, "../../");
+    const extensionTestsPath = path.resolve(__dirname, "./suite/index");
+
+    // Download VS Code, unzip it and run the integration test
+    await runTests({
+      extensionDevelopmentPath,
+      extensionTestsPath,
+      launchArgs: ["--disable-extensions"],
+    });
+  } catch (err) {
+    console.error("❌ Failed to run tests");
+    process.exit(1);
+  }
+}
+
+main().catch((err) => console.error(err));
```

Comment:
```suggestion
main();
```

From what I got on the code, you don't need that catch as it will never be called since the whole function is wrapped in a try/catch, and in the case of an error, the error is already logged and the process is exited right after.